system-parachains/asset-hubs/asset-hub-kusama/src/lib.rs

```diff
@@ -1345,6 +1343,8 @@ impl EnsureOriginWithArg<RuntimeOrigin, RuntimeParametersKey> for DynamicParamet
 		use crate::RuntimeParametersKey::*;
 
 		match key {
+			// TODO: @kianenigma/!bkontur - do we want to allow StakingAdmin here?
+			Inflation(_) => frame_system::ensure_root(origin.clone()),
```

Comment:
```suggestion
			Inflation(_) => frame_system::ensure_root(origin.clone()),
```
I think root is okay, there should only be changes needed to this rarely.

Reply:
True, inflation should be root-only, will fix.